src/docs/markdown/caddyfile/directives/basicauth.md

```diff
@@ -37,11 +37,30 @@ basicauth [<matcher>] [<hash_algorithm> [<realm>]] {
 
 ## Examples
 
-Protect all resources in /secret so only Bob can access them with the password "hiccup":
+Require authentication for all requests to `example.com`:
+
+```caddy
+example.com {
+	basicauth {
```

Comment:
I recall seeing `basicauth` being changed to `basic_auth` for Caddy 2.8 too.

I guess this feedback is better deferred to a follow-up PR (_or already planned_)

Reply:
Yep, deferred for 2.8